tensorboard/data/server/gen_protos_tool.rs

```diff
@@ -28,6 +28,7 @@ fn main() -> std::io::Result<()> {
     };
     tonic_build::configure()
         .out_dir(&out_dir)
+        .format(false) // don't run `rustfmt`; shouldn't be needed to build
```

Comment:
ah :) I knew something was weird about the generated file

Reply:
Yeah—I had `format(true)` (the default) originally, but that only worked
on my machine because I have `rustfmt` installed; it failed on Travis.
It would be nice for it to be formatted. We could format it pretty
easily if bazelbuild/rules_rust#388 were resolved, or we could copy what
bazelbuild/rules_rust#479 does and do it the hard way. I figured that to
start we could just accept the ugly generated code: we have the Rustdoc
server for normal API usage questions, and you can always run `rustfmt`
on it locally if you want to look at the implementation.